CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## v0.5.1-dev
 
 - Add `classical_delay` and `quantum_delay` as keyword arguments to the `RegisterNet` constructor to set a default global network edge latency.
+- Implement a feature to include a map as a background for plotting (`generate_map`).
```

Comment:
a minor point on technical writing style: "implement a feature to do x" is a bit wordy. "implement" already implies there is a "feature" and vice-versa. Something that more immediately gets to "do x" reads more naturally. You would notice though that I do not always remember to fix this style issue in my own writing